### PR TITLE
Docs: Improvements to installation and tutorial sections

### PR DESCRIPTION
**Pull request by manuelwedler**
## Description

This is part of solving https://github.com/raiden-network/team/issues/874. This applies several improvements to the installation guide and the getting started sections, including restructuring and textual enhancements. Furthermore this documents the UDC dialog in the webui tutorial and adds the custom token tutorial.

### CODE REVIEW

File: docs/custom-setups/custom_token.rst [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+   -  **initial_supply** **= 10000**
+   -  **decimal_units = 0**
+   -  **token_name = MyToken**
+   -  **token_symbol = TKN**
+
+5. Click the orange **"transact"** button.
+6. Confirm the transaction from MetaMask when prompted.
+
+In the Remix Ethereum IDE terminal you should now see an output similar to the picture below. 
+Congratulations, you have now successfully deployed a token to ethereum!
```
Comment (istankovic) [inline docs/custom-setups/custom_token.rst:102]: "ethereum" here should probably start with an uppercase letter to make it more consistent with the rest of the document..

File: docs/getting_started.rst [hunk 2]
```diff
@@ -3,24 +3,15 @@ Getting Started
 
 Now that your Raiden node is up and running, you can choose from the following tutorials, depending on your Raiden use case:
 
-Where do you want to use Raiden?
-================================
+See :doc:`Using Raiden on Mainnet <using-raiden-on-mainnet/overview>` for getting started to use Raiden in the Ethereum mainnet.
```
Comment (istankovic) [inline docs/getting_started.rst:6]: I'd rather use "on the Ethereum mainnet" than "in the Ethereum mainnet". The former phrasing also seems to be used elsewhere, e.g. 
```
Using Raiden on Mainnet Guide <using-raiden-on-mainnet/overview.rst>
```

Comment (manuelwedler) [reply docs/getting_started.rst:6]: Right, that would be consistent

File: docs/overview_and_guide.rst [hunk 3]
```diff
@@ -204,9 +212,10 @@ If problems arise for above method, please see `the Ropsten README <https://gith
 
 Then launch Raiden with the default testnet keystore path::
 
-    raiden --keystore-path  ~/.ethereum/testnet/keystore --pathfinding-service-address $PFS_ADDRESS
+    raiden --keystore-path  ~/.ethereum/testnet/keystore
 
-- **Using parity**
+Using parity
```
[the remark this answers is not in the document]
Comment (manuelwedler) [reply docs/overview_and_guide.rst]: I just did so :)

**Review by istankovic — APPROVED** [review]
Looks good to me.

**Pull request merged**